app/javascript/controllers/select_sibling_input_controller.js

```diff
@@ -0,0 +1,13 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = [ "radio", "checkbox" ]
+
+  selectRadio() {
+    this.radioTarget.checked = true
+  }
+
+  selectCheckbox() {
+    this.checkboxTarget.checked = true
+  }
+}
```

Comment:
We could try to design this helper to be even more extensible (say to make a sibling element required or not required).